controllers/controller.js

```diff
@@ -48,7 +49,33 @@ exports.getCommentsByID = (req, res, next) => {
 };
 
 exports.postCommentByID = (req, res, next) => {
-    const newComment = req.body
+  const newComment = req.body;
   const { review_id } = req.params;
-  postCommentByIDData(review_id, newComment);
+  const promises = [
+    postCommentByIDData(review_id, newComment),
+    checkIfCommentsExist(review_id),
+  ];
+  Promise.all(promises)
+    .then(([newComment]) => {
+      res.status(201).send(newComment);
+    })
+    .catch((err) => {
+      next(err);
+    });
+};
+
+exports.patchReviewByID = (req, res, next) => {
+  const updatedReviewBody = req.body;
+  const { review_id } = req.params;
+  const promises = [
+    patchReviewByIDData(review_id, updatedReviewBody),
+    checkIfCommentsExist(review_id),
```

Comment:
You don't need to keep adding checks to see if things exist for every end point. This is an expensive process in time and cash which should only be performed when necessary